# This contains the initial contact point and the desired contact point on the object in the object reference frame, with the pose in quaternion
# The output will be a 3 sequences of translations (identified by a sequence of points to define the contact, minus the initial one) and 3 sequences of rotations, and two regrasp points for the two grippers
# it also provides a sequence of distance variations between the translations
# The execution should perform each rotation before the translation

# The contact points are defined by a couple of points, in case of a parallel gripper

geometry_msgs/Pose[] initial_grasp
geometry_msgs/Pose[] desired_grasp
---
geometry_msgs/Point[] translations_1
geometry_msgs/Point[] translations_2
geometry_msgs/Point[] translations_3
float64[] rotations_1
float64[] rotations_2
float64[] rotations_3
float64[] distance_variations_1
float64[] distance_variations_2
float64[] distance_variations_3
geometry_msgs/Pose first_gripper_regrasp
geometry_msgs/Pose second_gripper_regrasp
uint16 master_index
